content/modules/guides/pages/create-business-logic-in-jmix.adoc

```diff
@@ -0,0 +1,157 @@
+:sample-project: jmix-business-logic-sample
+
+= Create Business Logic in Jmix
+
+One of the first questions after creating the initial data model in a Jmix application is: _Where should I put my custom logic?_ In any application, custom / business logic is the foundation that drives unique functionality, ensuring that user actions and data processing reflect the specific needs of the business. In Jmix, you have several options to structure your business logic, each suited for different types of operations and complexity levels.
+
+This guide will walk you through the available options, including placing logic in controllers, sharing logic across UI views, and centralizing business processes in reusable services. By the end of this guide, you will understand the pros and cons of each approach and how to implement them effectively in a Jmix application.
+
+
+== Requirements
+
+To follow this guide, you will need the following:
```

Comment:
I went for: "If you want to implement this guide step by step, you will need the following:", since it is actually only required if you don't want to only read it, but also code it.